samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -947,7 +960,8 @@
                 "ConsolidatedFilesOutputContainer": "[variables('ConsolidatedFilesOutputContainer')]",
                 "CreateAudioProcessedContainer": "[variables('CreateAudioProcessedContainer')]",
                 "AudioProcessedContainer": "[variables('AudioProcessedContainer')]",
-                "PiiCategories": "[variables('PiiCategories')]"
+                "PiiCategories": "[variables('PiiCategories')]",
```

Comment:
are all categories completely identical for both TA endpoints?

Reply:
no. But do we want to have a separate variable for pii and conversation pii for customers?

Reply:
Makes sense to use the variable for both, was just curious :D

Reply:
Keeping it as a separate variable but feel free to update this.